src/Middleware/WhoopsErrorResponseGenerator.php

```diff
@@ -91,11 +91,18 @@ private function prepareWhoopsHandler(ServerRequestInterface $request, PrettyPag
     {
         $uri = $request->getAttribute('originalUri', false) ?: $request->getUri();
         $request = $request->getAttribute('originalRequest', false) ?: $request;
+        
+        $serverParams = $request->getServerParams();
+        $scriptName = '';
+        
+        if (isset($serverParams['SCRIPT_NAME'])) {
+            $scriptName = $serverParams['SCRIPT_NAME'];
+        }
```

Comment:
Always found it bizarre that people think this is "cleaner". I didn't realise I had written really unwieldily code here. I guess the above is also using a ternary operator so it does at least make it consistent.